middle_end/flambda2/terms/flambda_primitive.mli

```diff
@@ -323,6 +323,8 @@ type unary_primitive =
   | End_region
       (** Ending delimiter of local allocation region, accepting a region name. *)
   | Obj_dup  (** Corresponds to [Obj.dup]; see the documentation in obj.mli. *)
+  | Get_header
+      (** get the header of a block; undefined behaviour if it's int *)
```

Comment:
I would say: "Get the header of a block.  This primitive is invalid code if provided with an immediate value."

Reply:
In addition let's explicitly say that the GC colour bits in the header should not be relied on as accurate.